### DISCUSSION TITLE
How to store values, and retrieve them without the 'comma' seperator?

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#853] [Q&A] [answered]
Hi

I'm trying to find out a way to store classes inside properties (to apply them later to an element)

I've done this [codepen](https://codepen.io/galinhalx/pen/yLvXPOR)

The issue I face, is about the commas that are retrieved by mavo, as soon as I have more that one stored value (normal behaviour)

Is there a way, or a trick to retrieve values without separated comma?

Or does someone have another trick in mind to store classes? Also tried to use mv-attr-class without success

**Comment by DmitrySharabin** [comment, marked as answer]
Hey there,

I would suggest using the `join()` function, like so: `join(list, " ")`.

https://mavo.io/docs/functions/#join

**Comment by GalinhaLX** [comment]
Thank you very much @DmitrySharabin 

It works with `join(list, ' ')`
as I was using it like `class="[join(list.property, ' ')]"`
